src/useBreakpoint.ts

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from 'react'
+
+function useBreakpoint(breakpoints: { [name: string]: number } = { laptopL: 1440, laptop: 1024, tablet: 768 }) {
```

Comment:
What do you think about a version where there would exist a factory function that creates the `useBreakpoint` hook?

```tsx
const useBreakpoint = createBreakpointHook({ laptopL: 1440, laptop: 1024, tablet: 768 });

const Demo = () => {
  const breakpoint = useBreakpoint();
};
```

Reply:
@streamich please, merge https://github.com/streamich/set-harmonic-interval/pull/4 and https://github.com/streamich/set-harmonic-interval/pull/3
Actually it is ASAP fixes, cuz now all IE support is broken.